src/FileConventions/Library.fs

```diff
@@ -0,0 +1,14 @@
+﻿module FileConventions
+
+open System.Linq
+open System.IO
+open System
+
+let HasShebang (fileInfo: FileInfo) =

```

Comment:
@realmarv if you included the logic for checking for the **correct** shebang in here then the function has to be renamed, otherwise it's confusing

Reply:
Done